Extension/src/nativeStrings.json

```diff
@@ -240,5 +240,6 @@
     "folder_tag": "Folder",
     "file_tag": "File",
     "compiler_default_language_standard_version_old" : "Compiler returned default language standard version: {0}. Since this version is old, will try to use newer version {1} as default.",
-    "unexpected_output_from_clang_tidy": "Unexpected output from clang-tidy: {0}. Expected: {1}."
+    "unexpected_output_from_clang_tidy": "Unexpected output from clang-tidy: {0}. Expected: {1}.",
+    "generaye_doxygen_comment":"Generate doxygen comment"
```

Comment:
Is "generaye_doxygen_comment" a title or command? If so, all words should be capitalized.

Reply:
I'm not sure, in this file, the first part is not capitalized

Reply:
Sorry, I meant if the string "Generate doxygen comment" for the UI is a title or not. I didn't meant to comment on the `generaye_doxygen_comment` part.